angular-workspace/projects/example-client-app/src/app/tabledemo/tabledemo.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component } from '@angular/core';
+import type { TableRecord } from '@ni/nimble-angular';
+
+interface SimpleTableRecord extends TableRecord {
+    stringValue1: string;
+    stringValue2: string;
+}
+
+@Component({
+    selector: 'example-tabledemo',
+    templateUrl: './tabledemo.component.html',
+    styleUrls: ['./tabledemo.component.scss']
+})
+export class TableDemoComponent {
+    public tableData: SimpleTableRecord[] = Array.from(Array(1000), (_, i) => {
+        return {
+            stringValue1: `Row  ${(i + 1)}`,
+            stringValue2: 'Example Text'
```

Comment:
nit: I'd vote to have both columns be dynamic since it makes scrolling performance a bit more obvious. But if having one of them be fixed data provides some value, I don't feel too strongly.

Reply:
Updated